src/glossary.rst

```diff
@@ -412,25 +397,21 @@ Glossary
       * :term:`job`
       * :term:`qualifier`
 
-   run directory
-      When a :term:`suite <Cylc suite>` is run a directory is created for all
-      of the files generated whilst the suite is running. This is called the
-      run directory and typically resides in the ``cylc-run`` directory:
-
-      ``~/cylc-run/<suite-name>``
+   workflow run directory
```

Comment:
Whilst "workflow run directory" is the full and proper name (or is it "flow run directory"?) we will want to refer to it throughout the docs as "run directory" or simply "run dir" if we can get away with that abbreviation (which I think we can if we link through to the glossary every now-and-again).

So here's my suggestion, differentiate the "run dir" and "cylc-run dir" with the word "cylc":

```
run dir
run directory
```

```
source dir
source directory
```

```
cylc-run
cylc-run directory
```

Thoughts?

Reply:
It is more concise, happy to change these.